### PR TITLE
Add Next Generation Passport recognition

### PR DESCRIPTION
**Pull request by andygonzalez2010**
https://travel.state.gov/content/travel/en/passports/passport-help/next-generation-passport.html

## Change Description

Adds Pattern to match next-generation U.S. passports

## Checklist

really simple pr

### CODE REVIEW

**Comment by omri374** [comment]
Thanks for the contribution! Could you please add one or more unit-tests?

File: presidio-analyzer/presidio_analyzer/predefined_recognizers/us_passport_recognizer.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class UsPassportRecognizer(PatternRecognizer):
     # Weak pattern: all passport numbers are a weak match, e.g., 14019033
     PATTERNS = [
         Pattern("Passport (very weak)", r"(\b[0-9]{9}\b)", 0.05),
+        Pattern("Passport Next Generation (very weak)", r"(\b[A-Z][0-9]{8}\b)", 0.1)
```
Comment (omri374) [inline presidio-analyzer/presidio_analyzer/predefined_recognizers/us_passport_recognizer.py:19]:
```suggestion
        Pattern("Passport Next Generation (very weak)", r"(\b[A-Z][0-9]{8}\b)", 0.05)
```

**Review by omri374 — APPROVED** [review]

**Comment by omri374** [comment]
@aflores-qb we would have to have the CLA signed prior to merging this. Would you be able to sign the CLA?

**Comment by andygonzalez2010** [comment]
> /azp run



> @aflores-qb we would have to have the CLA signed prior to merging this. Would you be able to sign the CLA?

I did, not sure why is not showing

**Comment by omri374** [comment]
> > /azp run
> 
> > @aflores-qb we would have to have the CLA signed prior to merging this. Would you be able to sign the CLA?
> 
> Hi sure, but I can't find how to do it

Could you please try the following command?
```
@microsoft-github-policy-service agree [company="{your company}"]
```

**Review by balteravishay — APPROVED** [review]
lgtm, thanks for the contribution @andygonzalez2010 !

**Comment by andygonzalez2010** [comment]
@microsoft-github-policy-service agree

**Pull request merged**